addons/binding/org.openhab.binding.yeelight/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,27 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: yeelight Binding
+Bundle-SymbolicName: org.openhab.binding.yeelight;singleton:=true
+Bundle-Vendor: openHAB
+Bundle-Version: 2.2.0.qualifier
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Bundle-ClassPath: .,
+ lib/yeelight_lib.jar,
+ lib/gson-2.7.jar
```

Comment:
Can you switch to the version which is already included by default, should be easy if you use no really specific functions.